docs/SDK/Explanation/authentication/sessionSigs/capability-objects.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 1
+---
```

Comment:
can remove the sidebar_position, since we are setting it in the `sidebarjs.js` file